diagnostic.md

```diff
@@ -15,6 +23,15 @@ at the `OL` DOM node).
 
 <!-- your answer starts here -->
 
+1.) You could use the class or id assigned to <li> as your selector.
+2.) You could use the class or id assigned to <ol> as your selector.
+
+You could also do the following:
+
+$("li")
+or
+$("ol")
```

Comment:
Good `$("li")` works but $("ol") would only great the entire list on a list element.